presto-main/src/main/java/io/prestosql/operator/scalar/VarbinaryFunctions.java

```diff
@@ -447,4 +447,24 @@ public static Slice rightPad(@SqlType("varbinary") Slice inputSlice, @SqlType(St
     {
         return pad(inputSlice, targetLength, padBytes, inputSlice.length());
     }
+
+    @Description("Reverse a given varbinary")
+    @ScalarFunction("reverse")
+    @SqlType(StandardTypes.VARBINARY)
+    public static Slice reverse(@SqlType("varbinary") Slice inputSlice)
+    {
+        if (inputSlice.length() == 0) {
+            return EMPTY_SLICE;
+        }
+        int length = inputSlice.length();
+        Slice reverse = Slices.allocate(length);
+        int forwardPosition = 0;
```

Comment:
I think we could simplify this as
```java
for (int i = 0; i < length; i++) {
    reverse.setByte(i, inputSlice.getByte(length - i));
}

Reply:
Yes, good point. I updated it to be more concise.